packages/block-library/src/embed/embed-preview.js

```diff
@@ -73,14 +73,11 @@ class EmbedPreview extends Component {
 		const { interactive } = this.state;
 
 		const html = 'photo' === type ? getPhotoHtml( preview ) : preview.html;
-		const parsedHost = new URL( url ).host.split( '.' );
-		const parsedHostBaseUrl = parsedHost
-			.splice( parsedHost.length - 2, parsedHost.length - 1 )
-			.join( '.' );
+		const embedSourceUrl = new URL( url ).hostname;
```

Comment:
I think we can use the [`getAuthority( url )`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-url/#getauthority) method from the URL package here.

You also need to update the e2e test for embedding to reflect the fixed basename.

![CleanShot 2024-07-03 at 09 33 46](https://github.com/WordPress/gutenberg/assets/240569/95bf7a33-c62f-43c7-92f1-452253e7eb50)

Reply:
Hi @Mamaduka,
I have used [getAuthority( url )](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-url/#getauthority) method and updated e2e test case. Please review it once.

Thank you.